README.md

```diff
@@ -1,5 +1,47 @@
-# Robusta
-Robusta is a troubleshooting and automations platform for Kubernetes. It streamlines everything that happens **after** you deploy your application. It is somewhat like Zapier/IFTTT for devops, with an emphasis on prebuilt useful automations and not just "build your own".
+<div align="center">
+  <p>
+    <a href="https://home.robusta.dev/" target="_blank">
```

Comment:
What do you think of reducing the height of the logo a little? It takes up a lot of sceenspace on desktop and pushes the content down below the fold.

Reply:
Reduced the size significantly.